packages/barracuda/0.1.0/dataset/waf/agent/stream/udp.yml.hbs

```diff
@@ -0,0 +1,3716 @@
+udp:
+host: "{{udp_host}}:{{udp_port}}"
+tags: {{tags}}
```

Comment:
See [comment](https://github.com/elastic/integrations/pull/186#discussion_r458441240) I made on the Suricata package, but in filebeat we pipe this to `toJSON`, so I'm assuming we need it rendered as a yaml array. The way this is right now it'll output as a comma-separated string.